eng/Publishing.props

```diff
@@ -19,7 +20,7 @@
       <ItemsToPushToBlobFeed Include="@(_InstallersToPublish)">
         <IsShipping>true</IsShipping>
         <PublishFlatContainer>true</PublishFlatContainer>
-        <RelativeBlobPath>$(_UploadPathRoot)/%(_InstallersToPublish.UploadPathSegment)/$(_PackageVersion)/%(Filename)%(Extension)</RelativeBlobPath>
+        <RelativeBlobPath>$(_UploadPathRoot)/%(_InstallersToPublish.UploadPathSegment)/%(Filename)%(Extension)</RelativeBlobPath>
```

Comment:
Not completely clear -- why do we need to delete the `_PackageVersion` here?

Reply:
Because it's a non-existent property, causing a `//` in the middle of the path. It eventually needs to be there, but that's a larger work item.